### PR TITLE
refac: small fix on table text padding

### PR DESCRIPTION
**Pull request by Cauatn**
Also made it better semantic in the header and footer, Lint automatically formatted for more consistent indentation, think not gonna make much difference tho.

Before: 
image

In PR: 
image

### CODE REVIEW

**Comment by ktz-dev** [comment]
Yes, this addresses the issue I just raised. #86 but it's still using paddings and margins to shift the things around to *look* centered. Read the suggested fix on my issue and let me know what you think.

**Comment by Cauatn** [comment]
i couldn't do it better :), ty for the comment.

now:
WhatsApp Image 2024-08-25 at 10 36 48
before:
WhatsApp Image 2024-08-25 at 10 36 32

**Comment by ktz-dev** [comment]
Hi. Wow! Thanks for that. 

If you remove the `min-w-[]` classes on the `<TableHead />` tags and set all three `<TableHead />` tags to `w-16` then you get the uniform column widths.

File: src/components/features.tsx [hunk 1]
```diff
@@ -230,92 +230,92 @@ export default function Features() {
                   <TableHead className="py-2 text-2xl font-bold text-black dark:text-white">
                     <span className="">How Zen compares to other browsers</span>
                   </TableHead>
-                  <TableHead className="py-2 font-bold text-center">
+                  <TableHead className="font-bold text-center w-8 px-0 min-w-[60px] mx-auto">
```
Comment (ktz-dev) [inline src/components/features.tsx:229]: Set width to `w-16` and `min-w-16`

File: src/components/features.tsx [hunk 2]
```diff
@@ -213,7 +213,7 @@ export default function Features() {
             />
           </div>
         </div>
-        <div className="flex flex-col p-5 md:p-16 md:col-span-2 border-t-2 row-span-2">
+        <div className="flex flex-col p-5 space-y-4 md:space-y-4 md:p-16 md:col-span-2 border-t-2 row-span-2">
```
Comment (ktz-dev) [inline src/components/features.tsx:212]: There is another `space-y-4` class without a breakpoint prefix and so `md:space-y-4` has no purpose.

File: src/components/features.tsx [hunk 3]
```diff
@@ -230,92 +230,92 @@ export default function Features() {
...
                       width={32}
                       src="/favicon.ico"
                       className="border-2 rounded-lg mx-auto mb-2 w-8 h-8"
                       alt="zen"
                     />
-                    Zen
+                    <span className="text-center">Zen</span>
                   </TableHead>
-                  <TableHead className="py-2 pl-4 lg:pr-0 pr-2 font-bold text-center opacity-60">
+                  <TableHead className="py-2 font-bold text-center w-8 px-0 min-w-[60px] mx-auto">
```
Comment (ktz-dev) [inline src/components/features.tsx:239]: Set width to `w-16` and `min-w-16`

File: src/components/features.tsx [hunk 4]
```diff
@@ -230,92 +230,92 @@ export default function Features() {
...
                       width={32}
                       src="/floorp.png"
                       className="bg-black dark:bg-white rounded-md p-1 mx-auto mb-2 w-7 h-7"
                       alt="floorp"
                     />
-                    Floorp
+                    <span className="text-center">Floorp</span>
                   </TableHead>
-                  <TableHead className="py-2 pl-2 font-bold text-center opacity-60">
+                  <TableHead className="py-2 font-bold text-center w-8 px-0 min-w-[75px] mx-auto">
```
Comment (ktz-dev) [inline src/components/features.tsx:249]: Set width to `w-16` and `min-w-16`

**Review by ktz-dev — COMMENTED** [review]
Sorry idk how to use GitHub.

**Review by ktz-dev — APPROVED** [review]
Look good to me 👍. Fixes #111 and #86. Just need to resolve merge conflicts. 

Screenshot 2024-08-29 at 9 05 44 AM

Screenshot 2024-08-29 at 9 06 11 AM

**Comment by ktz-dev** [comment]
They updated the features page. This table isn't there anymore.

**Comment by lawlesx** [comment]
> They updated the features page. This table isn't there anymore.

I think its better to close the issue and pr and not let it hang around

**Comment by Cauatn** [comment]
i agree

**Pull request closed without merging**